rtslib/tcm.py

```diff
@@ -840,6 +844,10 @@ def _get_hw_max_sectors(self):
         self._check_self()
         return int(self._parse_info('HwMaxSectors'))
 
+    def _get_max_data_area_mb(self):
+        self._check_self()
+        return int(self._parse_info('MaxDataAreaMB'))
+
```

Comment:
Andy,

I had one question/review comment for my own patch :)

Since this attribute/control setting is not added in all kernels this function will throw a AttributeError on older kernels. Is this the correct thing to do in python or should this function catch that exception and then return something like -1?

Reply:
Yeah, you could catch it and return None.